src/common.css

```diff
@@ -0,0 +1,64 @@
+.grid-layout {
+  display: grid;
+  grid-template-columns: 15rem 1fr;
+  grid-template-areas:
+    "header header header"
+    "aside main main";
+}
+
+.video-container {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1rem;
+}
+
+.navbar-bottom {
+  display: none;
+}
+
+@media only screen and (max-width: 945px) {
+  html {
+    font-size: 85%;
+  }
```

Comment:
Suggestion:- Can reduce the font size of the root instead of the entire html since I see you have used rems at many places.

Reply:
i guess html itself is root, m i right?

Reply:
The :root CSS pseudo-class matches the root element of a tree representing the document. In HTML, :root represents the <html> element and is identical to the selector html , except that its specificity is higher.

Reply:
So here yes, it does not matter much.